app/src/main/java/org/oppia/app/home/HomeFragmentController.kt

```diff
@@ -1,28 +1,44 @@
 package org.oppia.app.home
 
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
 import org.oppia.app.databinding.HomeFragmentBinding
 import org.oppia.app.fragment.FragmentScope
 import org.oppia.app.viewmodel.ViewModelProvider
 import org.oppia.domain.UserAppHistoryController
+import org.oppia.domain.exploration.ExplorationDataController
+import org.oppia.domain.exploration.TEST_EXPLORATION_ID_5
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.logging.Logger
 import javax.inject.Inject
 
+private const val TEST_EXPLORATION_ID = TEST_EXPLORATION_ID_5
+
 /** The controller for [HomeFragment]. */
 @FragmentScope
 class HomeFragmentController @Inject constructor(
```

Comment:
Change name to Presenter

Reply:
I will do that in a separate PR later. Mainly the code of this file will also get updated to check whether the fragment has already been loaded of not.

Reply:
Check this PR for these changes: #219